content/docs/start/experiment-management/experiments.md

```diff
@@ -269,8 +276,8 @@ into your `main` branch with the usual Git commands.
 ## Go Further
 
 There are many other features of `dvc exp`, like cleaning up unused experiments,
-sharing them without committing into Git, or getting differences between two
-experiments.
+saving the current workspace status as an experiment, sharing them without
```

Comment:
We've already mentioned saving so it's not "another feature" so no change needed here, I think.